### ISSUE TITLE
Investigate running the app on macOS

### ISSUE THREAD

**Issue by herzbube** [herzbube/littlego#394]
In Xcode on the "General" tab of the "Little Go" target there is a checkbox for "Mac Catalyst".

Reference: https://developer.apple.com/mac-catalyst/

**Comment by herzbube** [comment]
The current build apparently can already be downloaded, installed and executed on Silicon Macs. In the Mac App Store the app is listed under "iPhone & iPad apps".

What more does "Mac Catalyst" bring to an app?

**Comment by theophanemayaud** [comment]
I currently use it on a MacBook with M2, downloaded from the App Store, it runs great ! Maybe catalyst enables Intel Mac compatibility… ?

**Comment by herzbube** [comment]
@theophanemayaud That's very nice to hear! Until today I was not even aware that the app can be installed on desktop computers via the Mac App Store.

I now browsed through the "Mac Catalyst" introduction and the first tutorial steps. These are my takeaways:

- An app that adopts Mac Catalyst runs natively on macOS. Without Mac Catalyst the implication is that the app somehow runs via a compatibility layer, but I have not been able to find out any details on short notice.
- Mac Catalyst allows an app to be enhanced so that it has a better macOS user experience. The example given in the tutorial is to replace the tab bar with a sidebar.

These two points are already sufficient for me to decide that I will **not** adopt Mac Catalyst.

- Making the app run natively may require additional work to build libraries and frameworks being used by the app.
- Enhancing the app for macOS would require special code to be written for that platform.

In summary, Mac Catalyst means more work, so it's out for me. I am very glad that the app as it is runs out of the box on macOS, but I can't afford additional time to support another platform.
